.github/workflows/ci_weekly_report.yaml

```
@@ -0,0 +1,125 @@
+name: weekly CI test report
+on:
+  schedule:
+    - cron: '37 9 * * 1' # 9:37AM UTC -> 2:37AM PST every monday
+  workflow_dispatch:
+    inputs:
+      ci_runs:
+        default: '100'
+        description: 'The amount of runs to trigger in CI test report'
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  setup:
+    if: github.repository == 'commaai/openpilot'
+    runs-on: ubuntu-latest
+    outputs:
+      ci_runs: ${{ steps.ci_runs_setup.outputs.value }}
+    steps:
+      - id: ci_runs_setup
+        run: |
+          mylist="value=["
+
+          for i in $(seq 1 ${{ inputs.ci_runs }});
+          do
+              if [ $i != ${{ inputs.ci_runs }} ]; then
+                  mylist+="\"$i\", "
+              else
+                  mylist+="\"$i\""
+              fi
+          done
+          mylist+="]"
+
+          echo "$mylist" >> $GITHUB_OUTPUT
+      - run: |
+          echo "Number of CI runs for report: ${{ inputs.ci_runs }}"
+  ci_matrix_run:
+    needs: [ setup ]
+    strategy:
+      fail-fast: false
+      matrix:
+        value: ${{fromJSON(needs.setup.outputs.ci_runs)}}
+    uses: commaai/openpilot/.github/workflows/ci_weekly_run.yaml@master
+    with:
+      run_number: ${{ matrix.value }}
+
+  report:
+    needs: [ci_matrix_run]
+    runs-on: ubuntu-latest
+    if: always()
+    steps:
+      - name: Get job results
+        uses: actions/github-script@v7
+        id: get-job-results
+        with:
+          script: |
+            const jobs = await github
+              .paginate("GET /repos/{owner}/{repo}/actions/runs/{run_id}/attempts/{attempt}/jobs", {
+                owner: "commaai",
+                repo: "${{ github.event.repository.name }}",
+                run_id: "${{ github.run_id }}",
+                attempt: "${{ github.run_attempt }}",
+              })
+            var report = {}
+            jobs.slice(1, jobs.length-1).forEach(job => {
+              const jobName = job.name.split('/')[1].trim();
```

```suggestion
              const jobName = job.name.split('/')[2].trim();
```
Name field has an extra nesting `name: 'ci_matrix_run (1) / badges / create badges'`. Will fix the report missing most jobs.